automation/src/test/resources/diff-tests/application-default-credentials.ipynb

```diff
@@ -0,0 +1,76 @@
+{
+ "cells": [
```

Comment:
I wonder if I should verify the credentials from the Spark context as well..

Reply:
oh this is very exciting!

I'd be interested in confirming all our different credential needs - firecloud-py would be another example.  Doesn't have to be this PR.

Reply:
Yeah definitely.

I am also wondering if we could have test code interact directly with cells, instead of having to upload a notebook file. It'd be cool to write in your test:
```
executeCell("print credentials._service_account_email") shouldBe expectedSA
```